applications/nrf_desktop/sample.yaml

```diff
@@ -23,40 +24,56 @@ tests:
       - nrf5340dk/nrf5340/cpuapp
       - nrf54l15pdk/nrf54l15/cpuapp
       - nrf54l15pdk@0.3.0/nrf54l15/cpuapp
-    tags: bluetooth ci_build
+    tags: bluetooth ci_build sysbuild
   applications.nrf_desktop.zdebug_wwcb:
+    sysbuild: true
     build_only: true
     platform_allow: nrf52840dk/nrf52840
     integration_platforms:
       - nrf52840dk/nrf52840
-    tags: bluetooth ci_build
-    extra_args: CONF_FILE=prj_wwcb.conf
+    tags: bluetooth ci_build sysbuild
+    extra_args: FILE_SUFFIX=wwcb
   applications.nrf_desktop.zdebug_fast_pair.gmouse:
+    sysbuild: true
     build_only: true
     platform_allow: nrf52840dk/nrf52840 nrf52840gmouse/nrf52840
     integration_platforms:
       - nrf52840dk/nrf52840
       - nrf52840gmouse/nrf52840
-    tags: bluetooth ci_build
-    extra_args: CONF_FILE=prj_fast_pair.conf
+    tags: bluetooth ci_build sysbuild
+    extra_args: FILE_SUFFIX=fast_pair
                 FP_MODEL_ID=0x8E717D
                 FP_ANTI_SPOOFING_KEY=dZxFzP7X9CcfLPC0apyRkmgsh3n2EbWo9NFNXfVuxAM=
+                SB_CONFIG_BT_FAST_PAIR=y
+                SB_CONFIG_BOOTLOADER_MCUBOOT=y
+                SB_CONFIG_MCUBOOT_MODE_DIRECT_XIP=y
+                SB_CONFIG_SECURE_BOOT_APPCORE=n
+                SB_CONFIG_SECURE_BOOT=n
   applications.nrf_desktop.zdebug_mcuboot_qspi:
+    sysbuild: true
     build_only: true
     platform_allow: nrf52840dk/nrf52840
     integration_platforms:
       - nrf52840dk/nrf52840
-    tags: bluetooth ci_build
-    extra_args: CONF_FILE=prj_mcuboot_qspi.conf
+    tags: bluetooth ci_build sysbuild
+    extra_args: FILE_SUFFIX=mcuboot_qspi
+                SB_CONFIG_BOOTLOADER_MCUBOOT=y
+                SB_CONFIG_SECURE_BOOT_APPCORE=n
+                SB_CONFIG_SECURE_BOOT=n
   applications.nrf_desktop.zdebug_mcuboot_smp:
+    sysbuild: true
     build_only: true
     platform_allow: nrf52840dk/nrf52840 nrf52840gmouse/nrf52840
     integration_platforms:
       - nrf52840dk/nrf52840
       - nrf52840gmouse/nrf52840
-    tags: bluetooth ci_build
-    extra_args: CONF_FILE=prj_mcuboot_smp.conf
+    tags: bluetooth ci_build sysbuild
+    extra_args: FILE_SUFFIX=mcuboot_smp
```

Comment:
I think `SB_CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY=n` might be missing compared to the `prj_mcuboot_smp.conf`